sdk/monitor/Azure.Monitor.OpenTelemetry/src/Azure.Monitor.OpenTelemetry.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="OpenTelemetry.Extensions.AzureMonitor" VersionOverride="1.0.0-beta.3" />
```

Comment:
There are other 2 PRs does the change to `eng/Packages.Data.props`, will get this moved to props file in later commit.